django/test/client.py

```diff
@@ -309,6 +312,15 @@ def _encode_data(self, data, content_type):
                 charset = settings.DEFAULT_CHARSET
             return force_bytes(data, encoding=charset)
 
+    def _encode_json(self, data, encoder):
+        """
+        Return correctly encoded JSON if we have a dict.
```

Comment:
Avoid use of "we" in comments per our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).